### DISCUSSION TITLE
cloud migration

### DISCUSSION THREAD

**Discussion by ganesh-bala** [vishnu-ks/Cloud-migration#256] [Q&A]
Hi team,
                 it's a wonderful project and I have some questions.
                  1. in the migration of the data is in between store any cloud system.              
                  2. what is the downtime migrating the large volume?

**Comment by iamvishnuks** [comment]
Hi @ganesh-bala ,
Thanks for your feedback!
Xmigrate stores data in object storage of respective cloud and from there we import those disk images and build vms. 
We don't have to turn down the workload when the migration is happening, but there is a chance we may miss out some of the data which is written during the disk cloning process. Currently in the opensource version of xmigrate we don't capture the change data during migration. 
Major contributor for downtime is bandwidth that you can spare for the migration traffic between your datacenter and cloud and the size of the disks. To import disks to AWS it may take 20 mins, GCP and Azure have faster import speeds.
So consider time to transfer data + 20 mins as downtime.
